java-runtime/ftl-runtime/test-framework/src/main/java/xyz/block/ftl/java/test/internal/FTLTestResource.java

```diff
@@ -0,0 +1,27 @@
+package xyz.block.ftl.java.test.internal;
+
+import io.quarkus.test.common.QuarkusTestResourceLifecycleManager;
+
+import java.util.Map;
+
+public class FTLTestResource implements QuarkusTestResourceLifecycleManager {
+
+    FTLTestServer server;
+
+    @Override
+    public Map<String, String> start() {
+        server = new FTLTestServer();
+        server.start();
+        return Map.of("ftl.endpoint", "http://localhost:" + server.getPort());
```

Comment:
This is stupid but maybe use 127.0.0.1 instead because of the mac firewall? https://github.com/TBD54566975/ftl/pull/2298

Reply:
Ah if this is a client never mind.

Reply:
oops